bundles/org.openhab.binding.robonect/src/main/resources/OH-INF/config/config.xml

```diff
@@ -8,17 +8,17 @@
 	<config-description uri="channel-type:robonect:jobType">
 		<parameter name="remoteStart" type="text" pattern="^STANDARD|REMOTE_1|REMOTE_2$">
 			<label>Remote Start</label>
-			<description>The location to start the mowing job from.</description>
+			<description>The location to start the mowing job from. [STANDARD|REMOTE_1|REMOTE_2]</description>
```

Comment:
Perhaps something like this would be even better?
```suggestion
			<description>The location to start the mowing job from.</description>
			<options>
				<option value="STANDARD">Standard</option>
				<option value="REMOTE_1">Remote 1</option>
				<option value="REMOTE_2">Remote 2</option>
			</options>
```

Reply:
Thank you for your inputs - just got in touch with the code after using OH for some years now ;)

Reply:
Welcome as contributor. 🙂 Only one question left for the code: Is it safe to limit to those options or should it be possible to freely input other options not defined here?

Reply:
I now see the pattern already prevented other options, so the pattern can now be removed as restricted by options.

Reply:
I will do that in a minute.

Do you maybe know if it's intended behavour that the parameter.unit is not reflected in the UI?
```unit="m"``` indicates its minutes, but it's nowhere visible in the MainUI. Is it recommended to have a ```unitLabel``` despite having a unit defined?
The same applies for e.g. https://github.com/openhab/openhab-addons/blob/f08de2642973692b4f22cad007ff4eb27be076f1/bundles/org.openhab.binding.robonect/src/main/resources/OH-INF/thing/thing-types.xml#L72  - i needed to check the source to verify if its seconds or minutes because its not reflected in the UI.

Reply:
I'm not sure, but it might simply never have been implemented in MainUI. See also https://github.com/openhab/openhab-addons/pull/12461#discussion_r841053479.